pkg/operator/cluster/operator.go

```diff
@@ -80,8 +81,21 @@ func (c *clusterOperator) AddCluster(cluster *crv1.MySQLCluster) error {
 func (c *clusterOperator) UpdateCluster(newCluster *crv1.MySQLCluster) error {
 	newCluster.WithDefaults()
 
+	logging.LogCluster(newCluster).Debug("Updating services.")
+	err := c.updateServices(newCluster)
+	if err != nil {
+		logging.LogCluster(newCluster).WithField(
+			"error", err).Warn("Setting status")
+		setStateErr := c.setClusterState(
```

Comment:
Since we've started using kubernetes 1.10 this might be a good place to replace this function with the `UpdateStatus` for the `Status` subresource call?